service/controller/v13/encrypter/vault/vault.go

```diff
@@ -62,6 +62,8 @@ func NewEncrypter(c *EncrypterConfig) (*Encrypter, error) {
 
 		address: c.Address,
 		base:    base,
+		nonce:   "3c4caa13-dc08-f51f-51f8-02c4abb8d89f",
+		token:   "98b8b1aa-fa75-adba-7dd8-0499ba591bbd",
```

Comment:
Very naive q: is it safe to keep it in the code?

Reply:
Nope, just for testing purposes.

Reply:
Authentication from the container has been sorted, the hardcoded token has been removed.